bite-size-notes/week-8-e-Bayes Rule-RBE-RBF.md

```diff
@@ -0,0 +1,94 @@
+$$P(X|Y_1, Y_2) \propto P(Y_2|X,Y_1)P(X|Y_1)$$
+
+Keep in mind, we assume our measurements are independent:
+$$\propto P(Y_2|X)P(X|Y_1)$$
+$$\propto P(Y_2|X)P(Y_1|X)P(X)$$
+
+This applies for any number of measurements $P(X|Y_1,Y_2,\cdots, Y_N)$.
+
+## Recursive Bayes's Estimator
+
+We use the recursive Bayes's estimator to update the estimate of an unknown quantity as new data becomes available through the application of Bayes's Rule. It can be written as the following expression:
+$$P(X|Y_{1:K}) = \frac{P(Y_K|X)P(X|Y_{1:K-1})}{\int_{\tilde{X}} P(Y_K|\tilde{X})P(\tilde{X}|Y_{1:K-1})d\tilde{X}}$$
+
+A tilde is added to the integration term in the denominator in our notation for clarity. As well, note that the prior $P(X|Y_{1:K-1})$ for time step $k$ is simply the posterior from the previous time step $k-1$. Using this, we can then build the Recursive Bayesian Filter, but a few properties and mathematical expressions are worth going over first. The LHS of the RBE is proportional to the following expression:
+$$P(X|Y_{1:K}) \propto P(Y_{1:K}|X)P(X)$$
+where $P(Y_{1:K}|X)$ can be expanded via the chain rule to:
+$$P(Y_{1:K}|X) = P(Y_K|X,Y_{1:K-1})P(Y_{1:K-1}|X)$$
+
+Next, measurements with conditional independence w.r.t. x can be written as:
+$$P(Y_K|X)P(Y_{1:K-1}|X)$$
+
+Using this, we can then write the Bayes rule as the following (albeit gross but able to be simplified) expression:
+$$P(X|Y_{1:K})=P(Y_K|X)P(Y_{1:K-1}|X)P(X)$$
+$$P(X|Y_{1:K-1}) \propto P(Y_{1:K-1}|X)P(X)$$
+$$P(X|Y_{1:K}) = \frac{P(Y_K|X)\frac{P(Y_{1:K-1}|X)P(X)}{\int_X P(Y_{1:K-1}|X)P(X)dX}}{\int_X P(Y_K|X) \frac{P(Y_{1:K-1}|X)P(X)}{\int_X P(Y_{1:K-1}|X)P(X)dX} dX}$$
+
+Which can be simplified to:
+$$P(X|Y_{1:K}) = \frac{P(Y_K|X)\frac{P(Y_{1:K-1}|X)P(X)}{P(X|Y_{1:K-1})}}{\int_X P(Y_K|X) \frac{P(Y_{1:K-1}|X)P(X)}{P(X|Y_{1:K-1})} dX}$$
+
+### Independence
+
+Given random variables X and Y, $X\perp Y$ iff $P(X,Y)=P(X)P(Y)$
+$$X\perp Y \Rightarrow P(X|Y) = P(X)$$
+$$P(Y|X) = P(Y)$$
+
+### Conditional Independence
+
+Given measurements $Y_1$ and $Y_2$ which are conditionally independent given $X$
+$$P(Y_1|Y_2,X)=P(Y_1|X)$$
+$$P(Y_2|Y_1,X)=P(Y_2|X)$$
+
+## Recursive Bayesian Filter
+
+Say we have $X$ which changes over time and assuming the system is *Markov*, meaning "the next state depends on only the current state", we can write this as:
+$$P(X_0,X_1,X_2,\cdots,X_K) = P(X_0)\prod_{i=1}^KP(X_i|X_{i-1})$$
+
+However, this is the Markov chain without control. When we insert control, expressed as $P(X_K|X_{K-1}, U_{K-1})$ we get a *Markov Decision Process*. If we include observations and assume some/all of $X$ cannot be observed, we have a *Partially Observable Markov Decision Process*.
+
+The core of the Recursive Bayesian Filter is a two-step process: predict and update.
+
+### Predict
+
+$$P(X_K,X_{K-1}|Y_{1:K-1}) =  P(X_K|X_{K-1},Y_{1:K-1})P(X_{K-1}|Y_{1:K-1})$$
+$$P(X_K|X_{K-1})P(X_{K-1}|Y_{1:K-1})$$
```

Comment:
What is this expression saying? There's no relation here. Also, for some reason, this line doesn't show up in the markdown preview.

Reply:
Updated this expression to show in markdown. Also, I dropped an equals sign which is why that middle expression is floating there out of context.